.github/workflows/static-analysis.yml

```diff
@@ -81,8 +81,12 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v4
       - name: Check for Markdown issues
-        uses: DavidAnson/markdownlint-cli2-action@v15
+        run: |
+          npm install -g markdownlint-rule-max-one-sentence-per-line@0.0.2
```

Comment:
How about omitting the version here, as you also don't specify one for `markdownlint-cli2`? Or vice-versa, specify it for `markdownlint-cli2`.

Reply:
As the rule is a not very active one man project I wanted to avoid any issues here by using fixed version, for markdownlint itself I'm not so concerned about upgrades.

Reply:
Mind adding this rationale to the commit message then?